### PR TITLE
Run fast integration job on k3s

### PR DESCRIPTION
**Pull request by crabtree**
**Description**

Changes proposed in this pull request:

- Fast integration job on k3s

**Related issue(s)**

### CODE REVIEW

**Review by klaudiagrz — DISMISSED** [review]

File: prow/scripts/cluster-integration/kyma-integration-k3s.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+
+export KYMA_SOURCES_DIR="./kyma"
+export LOCAL_KYMA_DIR="./local-kyma"
+
+install::prereq() {
```
Comment (Ressetkk) [inline prow/scripts/cluster-integration/kyma-integration-k3s.sh:9]: This convention we use (or want to use) for the functions that are in the `lib` directory. Please change the functions names

Comment (crabtree) [reply prow/scripts/cluster-integration/kyma-integration-k3s.sh]: @Ressetkk ok, changed.

**Review by Ressetkk — APPROVED** [review]

**Review by klaudiagrz — APPROVED** [review]

**Pull request merged**